gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterConfigurationKeys.java

```diff
@@ -222,4 +222,11 @@ public class GobblinClusterConfigurationKeys {
   public static final String CONTAINER_ID_KEY = GOBBLIN_HELIX_PREFIX + "containerId";
 
   public static final String GOBBLIN_CLUSTER_SYSTEM_PROPERTY_PREFIX = GOBBLIN_CLUSTER_PREFIX + "sysProps";
+
+  public static final String HELIX_JOB_SCHEDULING_THROTTLE_ENABLED_KEY = "helix.job.scheduling.throttle.enabled";
+  public static final boolean DEFAULT_HELIX_JOB_SCHEDULING_THROTTLE_ENABLED_KEY = false;
```

Comment:
Not sure the use of word `throttle` captures what this key is actually used for. Phrases that make more sense to me
- Short circuit replanning
- Early exit
- Shortcut evaluation

Reply:
The reason we would call this `Throttle` is because we are preventing frequent replanner restarts from executing in the Gobblin AM. In other words, when a Helix workflow is computed by a regular start-up, replanner, or nurse replanner, there is no reason to ever recompute and resubmit.

This is because there is a significant "cold" start time to these workflows and there would be no benefit from allow these things to trigger shortly one after another.